.github/workflows/checks-workflows.yml

```diff
@@ -0,0 +1,31 @@
+name: Worflows check
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches:
+      - main
+
+env:
+  PYTHONUNBUFFERED: 1
+  FORCE_COLOR: 1
+  PYTHON_VERSION: "3.11"
+
+jobs:
+  lint-vizro-all:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python ${{ env.PYTHON_VERSION }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Workflows check
+        run: |
+          pip install pyyaml
+          cd .github/workflows
+          python ../../tools/yaml_check.py
```

Comment:
Let's pass in the directory to be checked as an argument instead - see comments in the python script.
```
python tools/yaml_check.py .github/workflow
```

Reply:
Done